### ISSUE TITLE
Feature request: add contract.skip support to skip entire test suites

### ISSUE THREAD

**Issue by travs** [trufflesuite/truffle#385]
## Issue

Can we add a `skip` feature to test  suites provided by `contract`?
The [truffle docs](http://truffleframework.com/docs/getting_started/javascript-tests) say
> What makes Truffle tests different from that of Mocha is the contract() function: This function works exactly like describe() except it enables Truffle's clean-room features.

Similar to #359, this is not what we would expect if `contract` is meant to be treated as `describe`.

## Steps to Reproduce

Ran `truffle test` with this code in one of my test files:

```js
contract.skip("CrowdSale", function(accounts){
  it("can sustain a large crowdsale");
})
```

## Expected Behavior

I expected the suite to skip, and for the other tests to run correctly.

## Actual Results

Process errored and exited:
```
TypeError: contract.skip is not a function
    at Object.<anonymous> (D:\SharedProjects\thing\test\crowdsale-test.js:141:10)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at C:\Users\Travis\AppData\Roaming\npm\node_modules\truffle\node_modules\mocha\lib\mocha.js:222:27
    at Array.forEach (native)
```

## Environment

* Operating System: Windows 7
* Truffle version: 3.2.1
* Ethereum client: TestRPC
* node version: 6.9.1
* npm version:  3.10.8

**Comment by travs** [comment]
The workaround for now that I am using is simply to wrap my test suite in a `describe` and continue as normally:
```js
describe.only("Fullscale test", function(){
  contract("CrowdSale", function(accounts){
    it("can sustain a large crowdsale");
  })
})
```

**Comment by cgewecke** [comment]
@travs This is available on 4.0.7 :) Thanks so much for suggesting it, hope you're well.
